diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/AvailableRulesDocTest.kt

```diff
@@ -0,0 +1,53 @@
+package org.cqfn.diktat.ruleset.utils
+
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.junit.Test
```

Comment:
```suggestion
import org.junit.jupiter.api.Test
```
Please change annotations to junit 5.